File: Practice/09/Python/09.py
h1, m1 = map(int, input().split(':'))
h2, m2 = map(int, input().split(':'))
import sys
if h1 > 23 or h1 < 0: sys.exit()
if h2 > 23 or h2 < 0: sys.exit()
if m1 > 59 or m1 < 0: sys.exit()
if m2 > 59 or m2 < 0: sys.exit()
h1 = h1 * 60
h2 = h2 * 60
m1 = m1 + h1
m2 = m2 + h2
if abs(m1 - m2) < 15: print("Встреча сотоится")
else: print("Встреча не состоится")

File: Practice/09/C++/09/09/09.cpp
#include <iostream>
using namespace std;

int main()
{
	setlocale(LC_ALL, "Rus");

	int h1, m1, h2, m2;
	char znak;

	cin >> h1 >> znak >> m1;
	cin >> h2 >> znak >> m2;

	if (h1 > 23 || h1 < 0) exit(0);
	if (h2 > 23 || h2 < 0) exit(0);
	if (m1 > 59 || m1 < 0) exit(0);
	if (m2 > 59 || m2 < 0) exit(0);

	h1 = 60 * h1;
	m1 = h1;
	h2 = 60 * h2;
	m2 = h2;

	if (abs(m1 - m2) > 15) cout << "Встреча не состоится";
	if (abs(m1 - m2) < 16) cout << "Встреча состоится";
}